modules/github_integration/db/migrate/20230316080525_add_repository_html_url.rb

```diff
@@ -0,0 +1,9 @@
+class AddRepositoryHtmlUrl < ActiveRecord::Migration[7.0]
+  def change
+    change_table :github_pull_requests, bulk: true do |t|
+      t.string :repository_html_url
+    end
+
+    execute "UPDATE github_pull_requests SET repository_html_url = CONCAT('https://github.com/', repository)"
```

Comment:
Is it always ok? What if some users are using the Enterprise version of GitHub?
Can `github_html_url` be used to find the right base url?

Something like this to test it:

```
select github_html_url, repository, position(repository IN github_html_url), substring(github_html_url from 0 to position(repository IN github_html_url)) from github_pull_requests;
```
so that request would become
```
execute "UPDATE github_pull_requests SET repository_html_url = CONCAT(substring(github_html_url from 0 to position(repository IN github_html_url)), repository)"
```

Reply:
It was the same for all PR's of instances I could find, but I'm fine with replacing it like you mentioned